Javascriptdemo2/demo3.js

```diff
@@ -0,0 +1,50 @@
+document.getElementById("submitBtn").addEventListener("click", () => {
+    const input = document.getElementById("TakingInput").value;
+    const number = parseInt(input);
+
+    if (number <= 1) {
+        console.log("It is neither prime or composite number");
+        return;
+    }
+
+    Prime1(number);
+});
+
+function prime(number) {
```

Comment:
Use arrow functions.  Here, you should use function name as isPrime. Use meaningful names.